.travis.yml

```diff
@@ -7,12 +7,12 @@ env:
   matrix:
     # Each line is a set of environment variables set before a build.
     # Thus each line represents a different build configuration.
-    - SHADERC_BUILD_TYPE=Release SHADERC_CODE_COVERAGE=OFF
-    - SHADERC_BUILD_TYPE=Debug SHADERC_CODE_COVERAGE=OFF
+    #- SHADERC_BUILD_TYPE=Release SHADERC_CODE_COVERAGE=OFF
+    #- SHADERC_BUILD_TYPE=Debug SHADERC_CODE_COVERAGE=OFF
     - SHADERC_BUILD_TYPE=Debug SHADERC_CODE_COVERAGE=ON
 
 compiler:
-  - clang
+  #- clang
```

Comment:
This doesn't seem necessary because the after_success action only runs for gcc. ?